Makefile

```diff
@@ -64,7 +64,7 @@ release:
 # Build used inside docker by 'release'
 release-docker-build:
 	export GOPATH=/go
-	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=$(CGO_ENABLED) GO111MODULE=on go build
+	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=$(CGO_ENABLED) GO111MODULE=on go build -buildvcs=false
```

Comment:
Hey @mathias-goebel - Your pr adjusts `make release-docker-build` however the issue refers to `make release`?

When I run `make release-docker-build` it works fine on my linux machine.  I do agree there is an issue with `make release` but that appears to be because the wrong docker image is being specified. I am looking into that.